services/prometheus/scrape_configs.go

```diff
@@ -174,68 +175,178 @@ func scrapeConfigForStandardExporter(interval time.Duration, node *models.Node,
 	return cfg, nil
 }
 
-func scrapeConfigForNodeExporter(interval time.Duration, node *models.Node, agent *models.Agent) (*config.ScrapeConfig, error) {
-	return scrapeConfigForStandardExporter(interval, node, nil, agent, []string{})
+func scrapeConfigForNodeExporter(s *models.MetricsResolutions, node *models.Node, agent *models.Agent) ([]*config.ScrapeConfig, error) {
```

Comment:
let's rename to scraperConfigsForNodeExporter

Reply:
and everywhere else where you changed return value from object to array

Reply:
I'm all for renaming `Config` to `Configs` but renaming `scrape` to `scraper` is a mistake (and possibly a typo) – https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config.
But it is not worth fixing now.

Reply:
I thought that Nurlan suggest `scraper` because the current function name sounds strange in English: `scrape config for node exporter` (But we don't scrape config here. We get config which called `scrapeConfig`). In this case, I would prefer the name like `get scrape config for node exporter`.

Reply:
https://golang.org/doc/effective_go.html#Getters

Reply:
Yep, but `scrape` is a verb in the context of Prometheus. Anyway, it does not make a lot of sense to argue about this.